__tests__/app.test.js

```diff
@@ -29,34 +29,116 @@ describe("GET /api/topics", () => {
         body.topics.forEach((topic) => {
           expect(topic).toMatchObject({
             slug: expect.any(String),
-            description: expect.any(String)
+            description: expect.any(String),
           });
         });
       });
   });
 });
 
 describe("GET /api/articles", () => {
-    it("200: responds with an array of article objects, each containing author, title, article id, topic, creation time, votes, and article image url properties. Each object should also have a comment count property, calculated using queries. The articles should be ordered by date in descending order", () => {
-      return request(app)
-        .get("/api/articles")
-        .expect(200)
-        .then(({ body }) => {
-          expect(body.articles).toBeInstanceOf(Array);
-          expect(body.articles).toBeSorted({key: 'created_at', descending: true});
-          expect(body.articles.length).toBeGreaterThan(0);
-          body.articles.forEach((article) => {
-            expect(article).toMatchObject({
-                article_id: expect.any(Number),
-                title: expect.any(String),
-                topic: expect.any(String),
-                author: expect.any(String),
-                created_at: expect.any(String),
-                votes: expect.any(Number),
-                article_img_url: expect.any(String),
-                comment_count: expect.any(Number)
-            });
+  it("200: responds with an array of article objects, each containing author, title, article id, topic, creation time, votes, and article image url properties. Each object should also have a comment count property, calculated using queries. The articles should be ordered by date in descending order", () => {
```

Comment:
just because you have mentioned in the test description "should also have" suggests that you could have written 2 tests here :)